### PR TITLE
Add ansible-operator-plugins

### PR DESCRIPTION
**Pull request by rashmigottipati**

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
/pj-rehearse pull-ci-openshift-ansible-operator-plugins-main-unit

File: ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  container:
+    from: src
+- as: sanity
+  commands: make test-sanity
+  container:
+    from: src
+- as: docs
+  commands: make test-docs
+  container:
+    from: src
```
[the remark this answers is not in the document]
Comment (rashmigottipati) [reply ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml]: Removed in latest commit

File: ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: openshift/Dockerfile
+  inputs:
+    ocp_4.16_base:
```
Comment (everettraven) [inline ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml:15]: it looks like there are some test failures related to "ocp_4.16_base" not existing which checks out. I think this either needs to be updated to "base" as that is an entry in the `base_images` field _OR_ the `base_images.base` definition should be updated to `base_images.ocp_4.16_base`

Comment (rashmigottipati) [reply ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml]: Updated in [commit](https://github.com/openshift/release/pull/48755/commits/6b96bbd112607db1aaa05bd72525674b7ddf86e3)

File: ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- as: all
+  commands: make test-all
+  container:
+    from: src
+- as: static
+  commands: make test-static
+  container:
+    from: src
+- as: sanity
+  commands: make test-sanity
```
Comment (grokspawn) [inline ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml:61]: Looks like you can use an approach like [this](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml#L85-L87) to setup what you need for linting.

**Pull request closed without merging**